Makefile

```diff
@@ -86,8 +86,6 @@ build-acceptance-tools:
 		echo "aborting: PROJECT_DIR not set";\
 	    exit 1;\
 	 fi
-	go build -o ${PROJECT_DIR}/mender-cli
```

Comment:
Why?

Reply:
because this would just rebuild what we already built in the compile stage and it is not what is published in the publish stage... so we are basically publishing untested binaries - which should not happen anywhere... but unfortunately it does. SO KILL IT WITH A CODE DELETION!